tests/components/august/test_binary_sensor.py

```diff
@@ -88,3 +88,18 @@
     assert binary_sensor_tmt100_name_online.state == STATE_OFF
     binary_sensor_tmt100_name_ding = hass.states.get("binary_sensor.tmt100_name_ding")
     assert binary_sensor_tmt100_name_ding.state == STATE_UNAVAILABLE
+
+
+async def test_doorbell_device_registry(hass):
+    """Test creation of a lock with doorsense and bridge ands up in the registry."""
+    doorbell_one = await _mock_doorbell_from_fixture(hass, "get_doorbell.offline.json")
+    doorbell_details = [doorbell_one]
+    await _create_august_with_devices(hass, doorbell_details)
+
+    device_registry = await hass.helpers.device_registry.async_get_registry()
+
+    reg_device = device_registry.async_get_device(
+        identifiers={("august", "tmt100")}, connections=set()
+    )
+    assert "hydra1" == reg_device.model
```

Comment:
Please always end with the expected value in test assertions.

Reply:
Fixed in https://github.com/home-assistant/home-assistant/pull/32197